Theme development/WP boilerplate.md

```diff
@@ -4,7 +4,7 @@ The repository can be found [here](https://github.com/infinum/wp-boilerplate).
 
 ## General info
 
-Infinum's WordPress boilerplate is an easy way to kick-start your WordPress development using modern programming practices. We wanted to utilize all advantages of the best PHP practices, which is why we used namespacing to ensure avoiding any possible class, method, and function clashes in the global namespace. We also implemented an autoloader, which means you no longer have to manually require your classes. Just instantiate them, and the autoloader will take care of the rest. With an added performance boost.
+Infinum's WordPress boilerplate is an easy way to kick-start your WordPress development using modern programming practices. We wanted to utilize all advantages of the modern PHP practices, which is why we used namespacing to ensure avoiding any possible class, method, and function clashes in the global namespace. We also implemented an autoloader, which means you no longer have to manually require your classes. Just instantiate them, and the autoloader will take care of the rest. With an added performance boost.
```

Comment:
Infinum's WordPress boilerplate is an easy way to kick-start your WordPress development using modern programming practices. We wanted to utilize all advantages of modern PHP practices, which is why we used namespacing to ensure avoiding any possible class, method, and function clashes in the global namespace. We also implemented an autoloader, which means you no longer have to manually require your classes. Just instantiate them, and the autoloader will take care of the rest. With an added performance boost.